custom/conf/app.example.ini

```diff
@@ -1337,10 +1337,10 @@ LEVEL = Info
 ;; Issue indexer storage path, available when ISSUE_INDEXER_TYPE is bleve
 ;ISSUE_INDEXER_PATH = indexers/issues.bleve ; Relative paths will be made absolute against _`AppWorkPath`_.
 ;;
-;; Issue indexer connection string, available when ISSUE_INDEXER_TYPE is elasticsearch or meilisearch
-;ISSUE_INDEXER_CONN_STR = http://elastic:changeme@localhost:9200
+;; Issue indexer connection string, available when ISSUE_INDEXER_TYPE is elasticsearch (e.g. http://elastic:password@localhost:9200) or meilisearch (e.g. http://:apikey@localhost:7700)
```

Comment:
Did I? 
<img width="742" alt="image" src="https://github.com/go-gitea/gitea/assets/9418365/ecb1def7-386d-41f9-b83c-5e969b2ed070">

I put an example URL for meilisearch because I didn't know where to set the API key unless I read the code.